pypeit/reduce.py

```diff
@@ -932,13 +942,16 @@ def show(self, attr, image=None, showmask=False, sobjs=None,
             mask_in = None
             bitmask_in = None
 
+        img_gpm = self.sciImg.boolean_mask(invert=True)
+
         if attr == 'global':
             # global sky subtraction
-            if self.sciImg.image is not None and self.global_sky is not None and self.sciImg.fullmask is not None:
+            if self.sciImg.image is not None and self.global_sky is not None \
+                    and self.sciImg.fullmask is not None:
                 # sky subtracted image
-                image = (self.sciImg.image - self.global_sky)*(self.sciImg.fullmask == 0)
-                mean, med, sigma = stats.sigma_clipped_stats(image[self.sciImg.fullmask == 0], sigma_lower=5.0,
-                                                       sigma_upper=5.0)
+                image = (self.sciImg.image - self.global_sky) * img_gpm.astype(float)
```

Comment:
Just a heads up that numpy automatically casts boolean masks to floats upon multiplication with a float array, but your way is more explicit.

Reply:
Fair point, but I'd prefer to leave this as explicit.